cms/djangoapps/contentstore/rest_api/v1/serializers.py

```diff
@@ -21,6 +21,21 @@ class LimitedProctoredExamSettingsSerializer(serializers.Serializer):
     enable_proctored_exams = serializers.BooleanField()
     proctoring_provider = serializers.CharField()
     proctoring_escalation_email = serializers.CharField(allow_blank=True)
+    create_zendesk_tickets = serializers.BooleanField(required=False)
+
+    def validate(self, data):
```

Comment:
I'm wondering if this is too much validation, as we don't enforce this on the frontend.

Reply:
I think this is good validation. Thank you for incorporate this into the business logic.